# This is the message format for AAUSHIP called faps
string DevID
string MsgID
string[] Data
float64 Time
